libs/domain/cart/summary/src/summary.component.ts

```diff
@@ -43,14 +42,22 @@ export class CartSummaryComponent extends CartComponentMixin<CartSummaryOptions>
     `;
   }
 
-  protected populateQuantity(
-    quantity: number | null,
-    options: CartSummaryOptions
-  ): string | number {
-    return quantity &&
-      options.maxVisibleQuantity &&
-      quantity > options.maxVisibleQuantity
-      ? `${options.maxVisibleQuantity}+`
-      : Number(quantity);
+  /**
+   * Renders the quantity of the total cart items.
+   *
+   * When the quantity is larger that the allowed max visible,
+   * the quantity is _truncated_ to the `maxVisibleQuantity`, e.g.
+   * "99+".
+   */
+  protected renderMark(): TemplateResult | void {
+    const quantity =
+      this.totalQuantity &&
+      this.componentOptions?.maxVisibleQuantity &&
+      +this.totalQuantity > this.componentOptions?.maxVisibleQuantity
+        ? `${this.componentOptions?.maxVisibleQuantity}+`
+        : Number(this.totalQuantity);
+    if (Number(this.totalQuantity)) {
```

Comment:
Why in one cases `+` in other `Number`?

Reply:
That's the design, where there are more than the configured number, we show "99+" in the cart summary.

Reply:
I am talking about these parts
<img width="586" alt="image" src="https://user-images.githubusercontent.com/16482929/227880618-f45d6be3-049c-457e-ab3f-22cb0c3e0369.png">